ideascube/serveradmin/templates/serveradmin/index.html

```diff
@@ -22,6 +22,7 @@
             <li>{% fa 'life-ring' 'fa-fw' %} <a href="{% url 'server:backup' %}">{% trans "Manage backups" %}</a></li>
             <li>{% fa 'plug' 'fa-fw' %} <a href="{% url 'server:battery' %}">{% trans "Monitor battery" %}</a></li>
             <li>{% fa 'wifi' 'fa-fw' %} <a href="{% url 'server:wifi' %}">{% trans "Manage Wi-Fi" %}</a></li>
+            <li>{% fa 'eye' 'fa-fw' %} <a href="{% url 'server:cards' %}">{% trans "Manage visible cards" %}</a></li>
```

Comment:
You use "visible" here, "displayed" on the card itself. Can we use the same in both places?